lib/recurly/venmo/strategy/braintree.js

```diff
@@ -65,11 +65,16 @@ export class BraintreeStrategy extends VenmoStrategy {
     braintree.client.create({ authorization }, (error, client) => {
       if (error) return this.fail('venmo-braintree-api-error', { cause: error });
       debug('Braintree client created');
-      braintree.venmo.create({ client, allowDesktop: true }, (error, venmo) => {
-        if (error) return this.fail('venmo-braintree-api-error', { cause: error });
-        debug('Venmo client created');
-        this.venmo = venmo;
-        this.emit('ready');
+      braintree.dataCollector.create({ client, paypal: true }, (error, collector) => {
+        if (error) return this.fail('venmo-braintree-api-error', { casue: error });
```

Comment:
There's a typo in this line.
`{ casue: error }` should be { cause: error }

Reply:
good catch!